github/orgs_actions_allowed.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2021 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+)
+
+// GetActionsAllowed gets the actions that are allowed in an organization.
+//
+// GitHub API docs: https://docs.github.com/en/rest/actions/permissions#get-allowed-actions-and-reusable-workflows-for-an-organization
+// Deprecated: please use `client.Actions.GetActionsAllowed` instead
```

Comment:
nit: I'll commit it - no action necessary.

```suggestion
// Deprecated: please use `client.Actions.GetActionsAllowed` instead.
```